src/main/java/algorithms/curated170/hard/CoinPath.java

```diff
@@ -0,0 +1,53 @@
+package algorithms.curated170.hard;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CoinPath {
+    public List<Integer> cheapestJump(int[] A, int B) {
+        int coins = A.length;
+        if (A == null || coins < 1 || A[coins - 1] < 0) {
+            return Collections.emptyList();
```

Comment:
```suggestion
            return List.of();
```
Maybe you prefer using the new and shorter way. 
There is no efficiency difference, but there is a subtle difference about serialization and exception handling that we needn't consider for this context.

Reply:
I chose `emptyList()` because it is easier to read. I didn't look up its technical properties though.

Reply:
ıt seems that `emptyList()` actually does a lot in  the background while `List.of()` does simpler operations. I will follow your advice.